website/src/components/Header/UserMenu.tsx

```diff
@@ -29,12 +37,20 @@ export function UserMenu() {
         desc: "Account Settings",
         icon: FiSettings,
       },
+    ];
+    const helpOptions = [
       {
         name: "Report a Bug",
         href: "https://github.com/LAION-AI/Open-Assistant/issues/new/choose",
         desc: "Report a Bug",
         icon: FiAlertTriangle,
       },
+      {
+        name: "FAQ",
+        href: "/faq",
+        desc: "Report a Bug",
+        icon: FiHelpCircle,
+      },
```

Comment:
do we have a FAQ page?

Reply:
We don't.  We can drop this entry for now.